editor/src/components/canvas/ui/floating-insert-menu.tsx

```diff
@@ -56,14 +56,13 @@ function convertInsertableComponentsToFlatList(
       options: componentGroup.insertableComponents.map(
         (componentToBeInserted, index): InsertMenuItem => {
           const source = index === 0 ? componentGroup.source : null
+          const label = componentToBeInserted.metadata?.originalName ?? componentToBeInserted.name
           return {
-            label: componentToBeInserted.name,
+            label,
```

Comment:
This is very minor, but we explicitly avoid the shorthand notation here, instead opting for `label: label`. We tend to favour an explicit coding style over an implicit one in general to minimise the risk of misunderstanding when someone unfamiliar with the code is reading it.